pfcpiface/up4.go

```diff
@@ -216,54 +207,46 @@ func (up4 *UP4) setupChannel() error {
 	return nil
 }
 
-func (up4 *UP4) initAllCounters() error {
+func (up4 *UP4) initAllCounters() {
 	log.Debug("Initializing counter for UP4")
 
-	err := up4.initCounter(preQosCounterID, "PreQosPipe.pre_qos_counter")
-	if err != nil {
-		return ErrOperationFailedWithReason("init preQosCounterID counter", err.Error())
-	}
+	counterID, counterSize := p4constants.CounterPreQosPipePreQosCounter, p4constants.CounterSizePreQosPipePreQosCounter
+	counterName := p4constants.GetCounterIDToNameMap()[counterID]
 
-	err = up4.initCounter(postQosCounterID, "PostQosPipe.post_qos_counter")
-	if err != nil {
-		return ErrOperationFailedWithReason("init postQosCounterID counter", err.Error())
-	}
+	up4.initCounter(preQosCounterID, counterName, counterSize)
 
-	return nil
+	counterID, counterSize = p4constants.CounterPostQosPipePostQosCounter, p4constants.CounterSizePostQosPipePostQosCounter
+	counterName = p4constants.GetCounterIDToNameMap()[counterID]
+
+	up4.initCounter(postQosCounterID, counterName, counterSize)
 }
 
-func (up4 *UP4) initMetersPools() error {
+func (up4 *UP4) initMetersPools() {
 	log.Debug("Initializing P4 Meters pools for UP4")
 
-	appMeter, err := up4.p4RtTranslator.getMeterByName(applicationMeter)
-	if err != nil {
-		return err
-	}
+	appMeterID, appMeterSize := p4constants.MeterPreQosPipeAppMeter, p4constants.MeterSizePreQosPipeAppMeter
```

Comment:
@osinstom as discussed on slack, meter size should be retrieve directly from the dataplane, not by the `p4info.txt` file that we have in `conf/` folder. How to proceed here? leave the refactor of meters out of this PR's scope?